doc/folders.md

```diff
@@ -426,23 +426,24 @@ Get All Cascade Policies on Folder
 ----------------------------------
 
 To get a list of all cascade policies on a folder, which show the metadata templates that are being applied to all 
-items in the folder, call [`getAll(BoxAPIConnection api, String folderID)`][get-all] with the ID of the folder.
+items in the folder, call [`getCascadePolicies()`][get-all] on that folder.
 
 ```java
 String folderID = "2222";
-BoxMetadataCascadePolicy.getAll(api, folderID);
+BoxFolder folder = new BoxFolder(api, folderID);
+Iterator<BoxMetadataCascadePolicy.Info> metadataCascadePolicies = folder.getCascadePolicies().iterator();
 ```
 
 You can also call [`getAll(BoxAPIConnection api, String folderID, String enterpriseID, int limit)`][get-all-with-limit] 
-and set the `enterpriseID` option to retrieve only cascade policies owned by a specific enterprise(defaults to the current enterprise).
+and set the `enterpriseID` option set to retrieve metadata cascade policies from another enterprise.
 
 ```java
 String folderID = "2222";
 String enterpriseID = "1234";
-Iterator<BoxMetadataCascadePolicy.Info> metadataCascadePolicies = BoxMetadataCascadePolicy.getAll(api, folderID, enterpriseID, 100);
+Iterator<BoxMetadataCascadePolicy.Info> metadataCascadePolicies = BoxMetadataCascadePolicy.getAll(api, folderID, enterpriseID, 100).iterator();
```

Comment:
This example should probably also use a for-in loop to capture the common use case